.gitignore

```diff
@@ -60,3 +60,9 @@ typings/
 
 # next.js build output
 .next
+
+# example config file
+examples/config/config.json
+
+# Example transcript output
+transcript.txt
```

Comment:
This would also be easier if we added an outputs folder :) we could just ignore everything inside it